src/MedPrep.Api/MedPrep.Api.csproj

```diff
@@ -19,6 +19,7 @@
     </PackageReference>
     <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="7.5.1" />
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="9.0.0-preview.3" />
+    <PackageReference Include="RestSharp" Version="112.0.0" />
```

Comment:
We already use the default http client that comes with .net. Is there a reason for using RestSharp? If there is we would need to refactor the client to use RestSharp instead of utilizing it directly in the service implementations.